components/camera/videosourcewrappers.go

```diff
@@ -95,6 +97,7 @@ func NewPinholeModelWithBrownConradyDistortion(pinholeCameraIntrinsics *transfor
 	return cameraModel
 }
 
+// NewVideoSourceFromReader is DEPRECATED!!! Please implement cameras according to the camera.Camera interface.
```

Comment:
How do the DEPRECATED comments work with staticcheck/linter in the RDK. Will linter fail or warn if code is added that uses the deprecated function?

Reply:
When the RPC or part of the RPC is deprecated in API - the linter will break, for this Deprecation string - I did not need to add points to the FFmpeg or webcam drivers to get the linter to pass, but that is based on rdk's tooling for its own linter. If a module has a more stringent linter that also checks the docstrings of the exported function somehow - I'd imagine that to break that go module's linter. I do not know if one such module exists.